charts/victoria-metrics-cluster/values.yaml

```diff
@@ -51,9 +51,13 @@ vmselect:
     # -- Image repository
     repository: victoriametrics/vmselect
     # -- Image tag
-    tag: v1.101.0-cluster
+    # override Chart.AppVersion
+    tag: ""
     # -- Image pull policy
     pullPolicy: IfNotPresent
+    # Variant of the image to use.
+    # e.g. cluster, enterprise-cluster
+    variant: "cluster"
```

Comment:
I would suggest treating `cluster` as a part of default tag and using `variant` to provide `-scratch` or `-enterprise-scratch` only.
This will be backward compatible as well as easier to use as user will only need to add the required prefix instead of populating an existing value and also adding suffix: `cluster-enterprise-scratch` vs `enterprise-scratch`

Reply:
We have two tags for cluster component, `v1.101.1-cluster` and `v1.101.1-enterprise-cluster`. So if `-cluster` is provided by `tag` instead of `variant`, user will have to update `tag` when they upgrade.

Reply:
Changed the default values, it's backward compatible now.